app/views/simplified-account/settings/email-notifications/collect-email-page.njk

```diff
@@ -0,0 +1,62 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Settings - Email notifications - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
+{% endblock %}
+
+{% block settingsContent %}
+
+    {{ govukBackLink({
+      text: "Back",
+      href: backLink
+    }) }}
+
+    <form id="collection-email-form" method="post" novalidate>
```

Comment:
again, be consistent with ids and names

Reply:
A lot of this nunjucks was copied from https://github.com/alphagov/pay-selfservice/blob/master/app/views/email-notifications/collection-email-mode.njk.

Reply:
Will rename to `email-collection-mode`